lib/api/v3/work_packages/work_packages_shared_helpers.rb

```diff
@@ -0,0 +1,99 @@
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'api/v3/work_packages/base_contract'
+require 'api/v3/work_packages/work_package_payload_representer'
+
+module API
+  module V3
+    module WorkPackages
+      module WorkPackagesSharedHelpers
+        extend Grape::API::Helpers
+        def request_body
+          env['api.request.body']
+        end
+
+        # merges the given JSON representation into @work_package
+        def merge_json_into_work_package!(work_package, json)
+          payload = WorkPackagePayloadRepresenter.create(work_package)
```

Comment:
````ruby
payload = ::API::V3::WorkPackages::WorkPackagePayloadRepresenter.create(work_package)
````

This would still fit within the line length limit and does not throw errors in development mode for me.

**I have no idea why this should be neccessary. Your way of writing it down should be exactly sufficient, but for some reason it is not...**